### DISCUSSION TITLE
《纯萌新如何逼疯开发者》

### DISCUSSION THREAD

**Discussion by KaSock** [Lost-MSth/Arcaea-server#25] [General]
教一下怎么搭建这个服务器...安卓11折腾了半天了，说明文档和指示在哪没看见在哪啊...

**Comment by KaSock** [comment]
![%1$Q_D07EIKQ) F DA`N8%I](https://user-images.githubusercontent.com/56071624/121483079-5cc29980-ca00-11eb-8ed0-015de962aa92.png)
这样不正常吧》。..

**Comment by Lost-MSth** [comment]
请求不合规，可能是https请求发到了http上导致的，不知道你干了什么------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年6月10日(星期四) 下午3:28
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;Re: [Lost-MSth/Arcaea-server] 如何使用啊... (#21)

**Comment by KaSock** [comment]
现在OK了，再就是看怎么操作手机上的东西了.....




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Lost-MSth/Arcaea-server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月10日(星期四) 下午3:31
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Lost-MSth/Arcaea-server] 如何使用啊... (#21)





 请求不合规，可能是https请求发到了http上导致的，不知道你干了什么------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 ***@***.***&amp;gt; 
 发送时间:&amp;nbsp;2021年6月10日(星期四) 下午3:28 
 ***@***.***&amp;gt;; 
 ***@***.***&amp;gt;; 
 主题:&amp;nbsp;Re: [Lost-MSth/Arcaea-server] 如何使用啊... (#21)

**Comment by KaSock** [comment]
服务器搭好了，但是就是手机arc访问不了这个本地服务器，用的还是官方的...

版本是3.5.1c的，文档里说的用Charles引导arc去访问本地服务器我不知道要怎么操作

手机WiFi也设置了代理

**Comment by Lost-MSth** [comment]
3.5.1应该没啥问题，按照操作还是比较简单的吧，不知道你证书设置有没有设置正确，确保抓到包应该就可以转发了
也可以参考baidu

**Comment by MBRjun** [comment]
我的手机Arcaea安装版本为3.5.2c（为了同时使用原版改了包名）
手机浏览器打开arcapi.lowiro.com正常解析到电脑
但是Arcaea似乎不认hosts直接用官方

**Comment by Lost-MSth** [comment]
能抓到包吗，看起来你解析成功了，可能是证书问题，也可能只需要重新开一下客户端就好了
等一下，你是改手机hosts的？那我不太清楚，dns解析我就没成功过，盲猜ssl pinning

**Comment by MBRjun** [comment]
手机Ping/浏览器访问都能指向电脑，但是游戏内连接至官服
请问大佬是用什么方法转发的请求？（

**Comment by Lost-MSth** [comment]
依赖那边写着的……
连到官服应该是SSL证书问题，然后变成socket连接了？我不清楚，真没试过（）

**Comment by KaSock** [comment]
> 3.5.1应该没啥问题，按照操作还是比较简单的吧，不知道你证书设置有没有设置正确，确保抓到包应该就可以转发了
> 也可以参考baidu

证书好像没设置。。不知道要怎么设置 : (

**Comment by Lost-MSth** [comment]
问baidu也行，我写的挺清楚的吧（
直接搜怎么HTTPS抓包就能找到相关教程

**Comment by KaSock** [comment]
哦，淦，看了下之前解决的issues，发现只有我一个人是心血来潮啥都不会的萌新，抓包什么的我也不知道抓了什么些鬼东西qwq
![image](https://user-images.githubusercontent.com/56071624/122795686-26cec080-d2f0-11eb-9bb1-373af740fa82.png)
这是我把手机上所有WiFi权限关了，只保留arcaea后抓的包，进入游戏到爬了个梯结算，不知道什么东西该留下来有用...

**Comment by KaSock** [comment]
你们说的数据库和后台又是什么东西qwq
![image](https://user-images.githubusercontent.com/56071624/122796051-8e850b80-d2f0-11eb-89da-de5f270ba18e.png)
![image](https://user-images.githubusercontent.com/56071624/122796078-95138300-d2f0-11eb-833e-c9b4262815d9.png)
是这个吗?

**Comment by Lost-MSth** [comment]
你把我弄傻了（）
不知道你在干什么
emmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm
这种基础的环境搭建问题，您应该尝试找一下教程的（我怎么记得我发过邮件给你啊，那里面提了教程的啊啊啊啊啊啊

**Comment by KaSock** [comment]
**《论如何逼疯开发者》**，就是不知道该干什么，找什么的教程呐qwq，邮件没有说教程

**Comment by KaSock** [comment]
莫西莫西，大佬还在吗qwq

**Comment by Lost-MSth** [comment]
wiki plz

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122798517-44515980-d2f3-11eb-812d-0d7c9a31610a.png)
阿巴阿巴阿巴

**Comment by Lost-MSth** [comment]
咳，请开动你的小脑筋可好（）
> 空白不是空白

**Comment by KaSock** [comment]
草，问题就是我直接看不懂这些东西，也不知道找哪些教程(废人啊这是

**Comment by Lost-MSth** [comment]
草，那几乎是手把手了吧啊吧啊吧(

**Comment by KaSock** [comment]
自信点，把几乎去掉XD
要不大佬我直接交学费吧qwq

**Comment by Lost-MSth** [comment]
别啊，有些四年级小朋友都会的()
请您加油

**Comment by KaSock** [comment]
你说的四年级小朋友是哪个，我这就去请教他qwq


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Lost-MSth/Arcaea-server"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年6月22日(星期二) 凌晨1:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Lost-MSth/Arcaea-server] 《纯萌新如何逼疯开发者》 (#21)





 
别啊，有些四年级小朋友都会的()
 请您加油

**Comment by Lost-MSth** [comment]
至少有两位，自己找吧()
而且是她不是他

**Comment by MBRjun** [comment]
> ![image](https://user-images.githubusercontent.com/56071624/122798517-44515980-d2f3-11eb-812d-0d7c9a31610a.png)
> 阿巴阿巴阿巴

这个是因为Github在中国大陆受到了审查。。。我现在不挂梯子都上不去Github

**Reply by Young-Lord** [reply]
怎么会是这个原因啊，明明是________，不信你可以在有网络环境的情况下去看看wiki页面

**Comment by KaSock** [comment]
好

**Comment by MBRjun** [comment]
我邮箱绑定了Github原来这就是我手机一直响的原因（

**Comment by KaSock** [comment]
> 我邮箱绑定了Github原来这就是我手机一直响的原因（

对不住，打扰大佬了😂

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122850659-337b0500-d340-11eb-9495-1f20312c6231.png)

大佬们，这样算是成功了嘛...但是进游戏没变化，版本是3.5.3c愚人节的，我电脑上的Charles需要调什么东西吗....

**Comment by MBRjun** [comment]
一样
改的Hosts？ Hosts对游戏无效 使用其他代理/转发

**Comment by KaSock** [comment]
> 一样
> 改的Hosts？ Hosts对游戏无效 使用其他代理/转发

大佬你是用什么代理的

**Comment by KaSock** [comment]
> 一样
> 改的Hosts？ Hosts对游戏无效 使用其他代理/转发

哦，淦，搞了半天游戏还是不鸟代理...我得用Charles干些啥吗

**Comment by KaSock** [comment]
我离成功应该就差这一步了...
![image](https://user-images.githubusercontent.com/56071624/122859635-e5b9c900-d34e-11eb-8b4c-c7304a021038.png)

**Comment by KaSock** [comment]
我试着用v2ray,然后发现我怎么调设置都不管用
![image](https://user-images.githubusercontent.com/56071624/122870437-400f5580-d360-11eb-95b0-a0d5eca07ef7.png)
命令行就跳这些
![image](https://user-images.githubusercontent.com/56071624/122870492-4e5d7180-d360-11eb-97a8-f98cc4c2c6f6.png)

**Comment by Lost-MSth** [comment]
400，有可能是https转发到http上的报错

**Comment by KaSock** [comment]
不会弄了，不知道问题出在哪。。不知道是不是setting的问题
![image](https://user-images.githubusercontent.com/56071624/122876113-b06da500-d367-11eb-88a0-37a6bf7ec10e.png)

**Comment by Lost-MSth** [comment]
不是，转发的问题罢了，我不知道v2ray能不能成功，毕竟没有试过（）

**Comment by KaSock** [comment]
大佬你们用什么转发的，我这v2ray我已经试了一个上午了qwq

**Comment by Lost-MSth** [comment]
写在readme里面了啊（）

**Comment by KaSock** [comment]
请问大佬用的哪个功能..远程映射还是端口转发还是别的什么qwq(拿到软件不会用

**Comment by Lost-MSth** [comment]
。。。。。map remote

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122885352-9769f180-d371-11eb-899b-0decd8498682.png)
![image](https://user-images.githubusercontent.com/56071624/122885552-c3857280-d371-11eb-90ae-307a49612031.png)

打咩达内~

**Comment by Lost-MSth** [comment]
抓包信息呢，这连到哪去了？

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122886378-7e157500-d372-11eb-9822-ec19e8152f3a.png)
![image](https://user-images.githubusercontent.com/56071624/122886445-8d94be00-d372-11eb-9fbe-20100bd8a46c.png)
![image](https://user-images.githubusercontent.com/56071624/122886463-92597200-d372-11eb-9c3f-60a6c63318c6.png)

**Comment by Lost-MSth** [comment]
你看下错误信息啊，八成证书没装()

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122892860-5aedc400-d378-11eb-898d-50142cf75dcf.png)
![image](https://user-images.githubusercontent.com/56071624/122892865-5c1ef100-d378-11eb-8ffd-1e530d090861.png)
emmmm，手机和电脑都安的一个证书，不知道怎么回事...
![image](https://user-images.githubusercontent.com/56071624/122892960-73f67500-d378-11eb-8880-4ed5c022c550.png)

**Comment by Lost-MSth** [comment]
ssl那边写*:*就行
安卓7以上有额外步骤，教程里写了()

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122894407-bbc9cc00-d379-11eb-8ff6-d9704edb025d.png)
这样吗

**Comment by Lost-MSth** [comment]
大概()

**Comment by KaSock** [comment]
G了，得解锁system分区，安卓11好像还不能qwq

**Comment by MBRjun** [comment]
什么设备

**Comment by KaSock** [comment]
> 什么设备

米10s

**Comment by MBRjun** [comment]
米10S解锁后可以将system挂载为rw

**Comment by YorokobiMaster** [comment]
11理论能解锁system分区吧，twrp都有了

**Comment by innocentvoice** [comment]
hmm i think we just need to change the sdk to 22 instead of 29 or 30,and the cert will work without root

**Comment by MBRjun** [comment]
> hmm i think we just need to change the sdk to 22 instead of 29 or 30,and the cert will work without root

降低安卓版本不现实 SDK22得安卓5了吧（？

**Comment by innocentvoice** [comment]
> > hmm i think we just need to change the sdk to 22 instead of 29 or 30,and the cert will work without root
> 
> 降低安卓版本不现实 SDK22得安卓5了吧（？

sounds like unrealistic but at least it worked

**Comment by innocentvoice** [comment]
just use apk editor

**Comment by KaSock** [comment]
> 米10S解锁后可以将system挂载为rw

什么操作那是qwq

**Comment by innocentvoice** [comment]
> > 米10S解锁后可以将system挂载为rw
> 
> 什么操作那是qwq

bootloader unlock 

must unlock the bootloader in order to root device

**Comment by YorokobiMaster** [comment]
话说各位，iOS设备怎么办，只能越狱装魔改ipa了吗？

**Comment by MBRjun** [comment]
> > 米10S解锁后可以将system挂载为rw
> 
> 什么操作那是qwq

将系统分区挂载为可读写
操作需要解锁
在解锁之前，请使用备份功能备份至OTG/电脑
解锁会清除全部数据

**Comment by MBRjun** [comment]
> 话说各位，iOS设备怎么办，只能越狱装魔改ipa了吗？

自己魔改IPA吧 最好有一台Mac设备
越狱可能会更方便

**Comment by Lost-MSth** [comment]
> 话说各位，iOS设备怎么办，只能越狱装魔改ipa了吗？

ios比安卓简单)

**Comment by KaSock** [comment]
突然就变成了群聊qwq

**Comment by MBRjun** [comment]
> 突然就变成了群聊qwq

Github群（确信）

**Comment by YorokobiMaster** [comment]
求给iOS指个路(
616这一更3.6，什么都不知道了，老方法全部木大(

**Comment by MBRjun** [comment]
> 求给iOS指个路(
> 616这一更3.6，什么都不知道了，老方法全部木大(

我有一台iOS设备和一台Mac 但是不太会改iOS的（

**Comment by Lost-MSth** [comment]
3.6？我也不清楚，问暗影去()

**Comment by MBRjun** [comment]
> 3.6？我也不清楚，问暗影去()

暗影只有B站有号吗（

**Comment by Lost-MSth** [comment]
我没有设备测试ios，所以根本没法搞，ios闪退都很正常)

**Comment by innocentvoice** [comment]
这已经成为github上的一个学习小组

**Comment by Lost-MSth** [comment]
> > 3.6？我也不清楚，问暗影去()
> 
> 暗影只有B站有号吗（

其它联系方式不清楚
(实际上他也会看issue的吧，可以蹲一蹲)

**Comment by TechnoHT** [comment]
can we put this into a GitHub *Discussion* instead of issue

**Comment by YorokobiMaster** [comment]
草

**Comment by innocentvoice** [comment]
lol

**Comment by YorokobiMaster** [comment]
为什么不呢（

**Comment by MBRjun** [comment]
> can we put this into a GitHub _Discussion_ instead of issue

没玩过，How 2 do it（

**Comment by innocentvoice** [comment]
> 没玩过，How 2 do it（

tell lost

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122903906-6e059180-d382-11eb-83cb-36b21540a786.png)
麻了

**Comment by MBRjun** [comment]
!(  )[https://cos.mbrjun.cn/IMGS/2021/06/22/l.jpg]
快要打骨折了

**Comment by MBRjun** [comment]
（MarkDown怎么引用图片来着

**Comment by MBRjun** [comment]
> ![image](https://user-images.githubusercontent.com/56071624/122903906-6e059180-d382-11eb-83cb-36b21540a786.png)
> 麻了

输入“su”，/system会被自动挂载为可读写（

**Comment by YorokobiMaster** [comment]
我寻思着
为啥不刷magisk

**Comment by YorokobiMaster** [comment]
刷个magisk多舒服，手机直接操作

**Comment by MBRjun** [comment]
他机子bl还没解吧（？

**Comment by KaSock** [comment]
> 我寻思着
> 为啥不刷magisk

有面具有root

**Comment by MBRjun** [comment]
MIUI是真的难用

**Comment by YorokobiMaster** [comment]
有面具你不早说啊喂（

**Comment by MBRjun** [comment]
> > 我寻思着
> > 为啥不刷magisk
> 
> 有面具有root

输完su那就已经是读写了，直接操作

**Comment by KaSock** [comment]
有面具怎么解锁，秋梨膏

> 有面具你不早说啊喂（

**Comment by MBRjun** [comment]
> 有面具怎么解锁，秋梨膏
> 
> > 有面具你不早说啊喂（

？你没解锁刷面具？

**Comment by innocentvoice** [comment]
> MIUI是真的难用

我同意

**Comment by KaSock** [comment]
输了su，mt管理器去操作还是显示读写失败

**Comment by YorokobiMaster** [comment]
不是，刷了面具，system分区不就应该也会解锁吗，，，

**Comment by KaSock** [comment]
> > 有面具怎么解锁，秋梨膏
> > > 有面具你不早说啊喂（
> 
> ？你没解锁刷面具？

我说的解锁system

**Comment by TechnoHT** [comment]
> > can we put this into a GitHub _Discussion_ instead of issue
> 
> 没玩过，How 2 do it（

go to Repository Settings
scroll down a bit you will see "Discussion" is unticked ( need access to the repo 1st )

**Comment by MBRjun** [comment]
> 输了su，mt管理器去操作还是显示读写失败

（Read Only File System Error？）

**Comment by KaSock** [comment]
> 不是，刷了面具，system分区不就应该也会解锁吗

难不成要换个文件管理器

**Comment by innocentvoice** [comment]
> > > can we put this into a GitHub _Discussion_ instead of issue
> > 
> > 
> > 没玩过，How 2 do it（
> 
> go to Repository Settings
> scroll down a bit you will see "Discussion" is unticked ( need access to the repo 1st )

only the owner can do it

**Comment by MBRjun** [comment]
> > 不是，刷了面具，system分区不就应该也会解锁吗
> 
> 难不成要换个文件管理器

我MT管理器能操作system，实在不行twrp下（

**Comment by KaSock** [comment]
> > > 不是，刷了面具，system分区不就应该也会解锁吗
> > 
> > 
> > 难不成要换个文件管理器
> 
> 我MT管理器能操作system，实在不行twrp下（

这机子没twrp，惊不惊喜

**Comment by MBRjun** [comment]
> > > > 不是，刷了面具，system分区不就应该也会解锁吗
> > > 
> > > 
> > > 难不成要换个文件管理器
> > 
> > 
> > 我MT管理器能操作system，实在不行twrp下（
> 
> 这机子没twrp，惊不惊喜

机子设备代号给下我找找
（真没有也没啥救 我记得有小米专门解锁/system的工具

**Reply by KaSock** [reply]
M2102J2SC
THYME

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122904861-4f53ca80-d383-11eb-83e5-a13afc49c196.png)
KUSO！

**Comment by innocentvoice** [comment]
请注意，只要使用低于v3.6.0的版本，就必须将GAME_API_PREFIX改为/latte/13而不是/blockchain/14。

**Comment by MBRjun** [comment]
好活

**Comment by YorokobiMaster** [comment]
来力(

**Comment by KaSock** [comment]
好家伙，讨论区创立了

**Comment by YorokobiMaster** [comment]
顺便有twrp

**Comment by innocentvoice** [comment]
现在我们可以在这里学习了

**Comment by MBRjun** [comment]
![我快骨折了](https://cos.mbrjun.cn/IMGS/2021/06/22/l.jpg)

**Reply by innocentvoice** [reply]
RIP

**Reply by YorokobiMaster** [reply]
草（

**Reply by MBRjun** [reply]
我引用我自己网站上的图居然会被Github二次缓存

**Reply by TechnoHT** [reply]
average GitHub discussion 😂

**Comment by YorokobiMaster** [comment]


**Reply by KaSock** [reply]
啊，这个残心的...

**Reply by KaSock** [reply]
他这个线刷的rec我不知道要刷哪去，我ab分区的机子不知道该往哪刷qwq

**Reply by MBRjun** [reply]
小米机子有AB分区？

**Reply by YorokobiMaster** [reply]
不是直接刷吗，，，淦

**Reply by KaSock** [reply]
他这个要原始boot的..我修补了面具的boot好像刷不了这个rec了，淦

**Reply by YorokobiMaster** [reply]
噗。。。。。。你刷的官改还是啥，包里面不就有boot

**Reply by MBRjun** [reply]
> 他这个要原始boot的..我修补了面具的boot好像刷不了这个rec了，淦

第一步刷个别的好用的系统，第二步别碰MIUI（

**Reply by KaSock** [reply]
> 噗。。。。。。你刷的官改还是啥，包里面不就有boot

官方包修补boot刷面具的

**Reply by KaSock** [reply]
> > 他这个要原始boot的..我修补了面具的boot好像刷不了这个rec了，淦
> 
> 第一步刷个别的好用的系统，第二步别碰MIUI（

好家伙，这个机子也没类原生之类的系统，顶天一个官改

**Comment by MBRjun** [comment]
![90SVV3R FW87H0)%ICX808O](https://user-images.githubusercontent.com/55020690/122905779-18ca7f80-d384-11eb-87d9-be77c56e28a1.png)
上午在直播越级
越级不好，不要越

**Reply by KaSock** [reply]
这不来个直播间地址

**Reply by MBRjun** [reply]
https://live.bilibili.com/21610293
暑假再播，快考试了

**Comment by MBRjun** [comment]
翡翠鸡是啥玩意 解锁ES要打10+ 阴间
（刚才回复失败了刷新没这评论了

**Reply by YorokobiMaster** [reply]
这个时候解收益不大的，真的（
prs打烂再说（

**Reply by MBRjun** [reply]
prsES AA/F 骨折A/HC（
最近在推7.5

**Comment by KaSock** [comment]
我还是去找找怎么直接解system吧，这个rec可能会翻车

**Reply by YorokobiMaster** [reply]
miui找个能用的版本也行
但是10s不知道类原生有没有official包，反正我的mix3有official（

**Reply by MBRjun** [reply]
我之前刷机 然后脑抽在刷机之前三清
刷的时候设备信息被清掉了 校验失败
REC被刷没了 MIFlash线刷救砖的

**Reply by YorokobiMaster** [reply]
设备信息清掉？那不删了机型验证再刷嘛

**Reply by MBRjun** [reply]
> miui找个能用的版本也行
> 但是10s不知道类原生有没有official包，反正我的mix3有official（

我设备用的刷机包 //https://download.mbrjun.cn/%E5%88%B7%E6%9C%BA%E5%8C%85/Pixel%20Experience/%E5%B0%8F%E7%B1%B3/Mi%209%20SE%5Bgrus%5D/PixelExperience_Plus_grus-10.0-20200920-1505-OFFICIAL.zip

**Reply by YorokobiMaster** [reply]
？pe属是羡慕了

**Reply by KaSock** [reply]
哦，淦，不是管理器的问题，还是得解锁系统

**Reply by MBRjun** [reply]
> 哦，淦，不是管理器的问题，还是得解锁系统

我这里Github被屏蔽 得挂梯子上

**Comment by MBRjun** [comment]
看见有个 Create issue from discussion按钮，搁这套娃：discussion issue互转

**Comment by KaSock** [comment]
草，兄弟们，我歇逼了

**Reply by KaSock** [reply]
重启自动进fastboot是什么鬼

**Reply by MBRjun** [reply]
> 重启自动进fastboot是什么鬼

您的系统和Rec都刷炸了就容易这样

**Reply by KaSock** [reply]
啊这，我是刷了那个搞机助手的禁用DM验证

**Reply by KaSock** [reply]
啊，这咋整，好家伙，明天还得上班打卡，歇逼了哈哈哈

**Reply by MBRjun** [reply]
> 啊，这咋整，好家伙，明天还得上班打卡，歇逼了哈哈哈

手动重启也只会进入Fastboot？

**Reply by KaSock** [reply]
昂

**Reply by MBRjun** [reply]
> 昂

你的全部数据已丢失
使用线刷完整包救砖

**Reply by KaSock** [reply]
不至于，我保留数据线刷试试

**Reply by MBRjun** [reply]
> 不至于，我保留数据线刷试试

选择保留数据或许有救
但是我的PE系统没线刷包

**Comment by Lost-MSth** [comment]
所以，问题是你们在干啥嘛

既不用root也不用越狱更不用刷机的啊()

**Reply by KaSock** [reply]
作qwq

**Reply by KaSock** [reply]
主要还是那个证书得丢根证书里面去才能ssl代理到，然后要丢根目录就得system可读写，要他可读写我就得刷机，然后我就歇逼了qwq

**Reply by MBRjun** [reply]
> 主要还是那个证书得丢根证书里面去才能ssl代理到，然后要丢根目录就得system可读写，要他可读写我就得刷机，然后我就歇逼了qwq

我记得小米有system解锁工具以前用过

**Reply by KaSock** [reply]
那个是在没有面具的情况下的，有面具那个工具明确说了用不了..

**Reply by MBRjun** [reply]
> 那个是在没有面具的情况下的，有面具那个工具明确说了用不了..

你用的哪个

**Reply by KaSock** [reply]
酷安那个，打开就是绿色界面中间一个开关的

**Reply by Lost-MSth** [reply]
亲，安卓7以上解决方法还挺多的，你偏要选根证书的方法干啥，我教程里写了的啊，之前也有夜架yuuuuue提过的不是吗，降个targetsdkversion就行（

**Reply by KaSock** [reply]
我其实一直没找到你说的教程和指示文档在哪...

**Reply by Lost-MSth** [reply]
哦，太好了，看起来我藏的还可以（

**Reply by MBRjun** [reply]
> 哦，太好了，看起来我藏的还可以（

大陆特色屏蔽和打不开

**Reply by Lost-MSth** [reply]
不是屏蔽

**Reply by KaSock** [reply]
好家伙，我居然离成功就差一个证书信任

**Reply by KaSock** [reply]
今天晚上我手机应该能恢复正常，届时直接去搞JustTrustMe模块去莽，然后明天安然无恙的去上班(美妙

**Reply by KaSock** [reply]
卡米了，好家伙，只能清除全部数据了...

**Reply by MBRjun** [reply]
搞机不备份，真男人

**Reply by KaSock** [reply]
> 搞机不备份，真男人

有个5.18的备份在电脑里，问题不大，虽然很多东西都无了qwq

**Reply by KaSock** [reply]
> 搞机不备份，真男人

备份我推荐再买个小米手机，用小米快传备份，数据还能保留下了

**Reply by MBRjun** [reply]
> > 搞机不备份，真男人
> 
> 备份我推荐再买个小米手机，用小米快传备份，数据还能保留下了

MIUI自带备份不能用？

**Reply by KaSock** [reply]
> > > 搞机不备份，真男人
> > 
> > 
> > 备份我推荐再买个小米手机，用小米快传备份，数据还能保留下了
> 
> MIUI自带备份不能用？

那个不保留应用数据，有时候挺麻烦的

**Reply by MBRjun** [reply]
> > > > 搞机不备份，真男人
> > > 
> > > 
> > > 备份我推荐再买个小米手机，用小米快传备份，数据还能保留下了
> > 
> > 
> > MIUI自带备份不能用？
> 
> 那个不保留应用数据，有时候挺麻烦的

自带备份是保留应用数据的，可以用来修改存档（bushi

**Reply by KaSock** [reply]
只保留个应用，就相当于给你备份了安装包

**Reply by KaSock** [reply]
![image](https://user-images.githubusercontent.com/56071624/122915962-ee31f400-d38e-11eb-848b-ac002ac52e58.png)
勇

**Reply by MBRjun** [reply]
> 只保留个应用，就相当于给你备份了安装包

带 /data/app 的数据啊

**Reply by KaSock** [reply]
> > 只保留个应用，就相当于给你备份了安装包
> 
> 带 /data/app 的数据啊

data/app都是安装包啊，数据什么的都在/data/data还有内部储存/Android/data里面呐

**Reply by MBRjun** [reply]
> > > 只保留个应用，就相当于给你备份了安装包
> > 
> > 
> > 带 /data/app 的数据啊
> 
> data/app都是安装包啊，数据什么的都在/data/data还有内部储存/Android/data里面呐

我的就带应用数据（
可能是MIUI版本问题

**Reply by MBRjun** [reply]
> > > 只保留个应用，就相当于给你备份了安装包
> > 
> > 
> > 带 /data/app 的数据啊
> 
> data/app都是安装包啊，数据什么的都在/data/data还有内部储存/Android/data里面呐

自己打data镜像不是没手就行

**Reply by KaSock** [reply]
> > > > 只保留个应用，就相当于给你备份了安装包
> > > 
> > > 
> > > 带 /data/app 的数据啊
> > 
> > 
> > data/app都是安装包啊，数据什么的都在/data/data还有内部储存/Android/data里面呐
> 
> 自己打data镜像不是没手就行

草

**Reply by KaSock** [reply]
之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人

**Reply by MBRjun** [reply]
> 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人

MIUI残缺版root管理请（

**Reply by KaSock** [reply]
> > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> 
> MIUI残缺版root管理请（

开发版吗，饶了我吧，我现在还没900分呢

**Reply by MBRjun** [reply]
> > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > 
> > 
> > MIUI残缺版root管理请（
> 
> 开发版吗，饶了我吧，我现在还没900分呢

我500分都能用内测版

**Reply by KaSock** [reply]
> > > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > > 
> > > 
> > > MIUI残缺版root管理请（
> > 
> > 
> > 开发版吗，饶了我吧，我现在还没900分呢
> 
> 我500分都能用内测版

那没事了，偷渡滴神

**Reply by MBRjun** [reply]
> > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > 
> > 
> > MIUI残缺版root管理请（
> 
> 开发版吗，饶了我吧，我现在还没900分呢

你不会去MiFlashPro去下开发板线刷包吗

**Reply by MBRjun** [reply]
> > > > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > > > 
> > > > 
> > > > MIUI残缺版root管理请（
> > > 
> > > 
> > > 开发版吗，饶了我吧，我现在还没900分呢
> > 
> > 
> > 我500分都能用内测版
> 
> 那没事了，偷渡滴神

老机型一般直接过没啥要求

**Reply by KaSock** [reply]
> > > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > > 
> > > 
> > > MIUI残缺版root管理请（
> > 
> > 
> > 开发版吗，饶了我吧，我现在还没900分呢
> 
> 你不会去MiFlashPro去下开发板线刷包吗

好家伙，现在刷了清除数据的卡第二屏

**Reply by MBRjun** [reply]
> > > > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > > > 
> > > > 
> > > > MIUI残缺版root管理请（
> > > 
> > > 
> > > 开发版吗，饶了我吧，我现在还没900分呢
> > 
> > 
> > 你不会去MiFlashPro去下开发板线刷包吗
> 
> 好家伙，现在刷了清除数据的卡第二屏

？试试刷上TWRP，然后卡刷MIUI

**Reply by MBRjun** [reply]
> > > > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > > > 
> > > > 
> > > > MIUI残缺版root管理请（
> > > 
> > > 
> > > 开发版吗，饶了我吧，我现在还没900分呢
> > 
> > 
> > 你不会去MiFlashPro去下开发板线刷包吗
> 
> 好家伙，现在刷了清除数据的卡第二屏

对了，MIUI部分机型有防回刷

**Reply by KaSock** [reply]
> > > > > > 之前不知道为什么修补boot刷面具的时候把基带刷没了，就很搞人
> > > > > 
> > > > > 
> > > > > MIUI残缺版root管理请（
> > > > 
> > > > 
> > > > 开发版吗，饶了我吧，我现在还没900分呢
> > > 
> > > 
> > > 你不会去MiFlashPro去下开发板线刷包吗
> > 
> > 
> > 好家伙，现在刷了清除数据的卡第二屏
> 
> 对了，MIUI部分机型有防回刷

活了，机子开了，数据全无，我的学习资料啊——

**Reply by KaSock** [reply]
好了，直接开始恢复root吧...
![image](https://user-images.githubusercontent.com/56071624/122926758-c6e12400-d39a-11eb-8273-bceeeeb2ba56.png)

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/122942818-0f9fd980-d3a9-11eb-9193-a8d539170890.png)
成功l了！，泪目了家人们
![image](https://user-images.githubusercontent.com/56071624/122942877-1e868c00-d3a9-11eb-8318-7412b63d0f75.png)

**Reply by YorokobiMaster** [reply]
羡慕死了(

**Comment by KaSock** [comment]
好家伙，我直接好家伙，终于可以躺下睡觉去了 ------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年6月22日(星期二) 晚上11:41
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Lost-MSth/Arcaea-server] 《纯萌新如何逼疯开发者》 (#25)

**Comment by KaSock** [comment]
emmmm.3.6以上的客户端需要做什么操作，好像改sdk到22也没用了...

**Reply by Lost-MSth** [reply]
看一下之前的某些issue里提过，要干掉ssl证书校验

**Reply by KaSock** [reply]
> 看一下之前的某些issue里提过，要干掉ssl证书校验

偶淦，这就涉及到我知识盲区了

**Comment by KaSock** [comment]
![image](https://user-images.githubusercontent.com/56071624/123098816-1b9ca180-d464-11eb-850c-0310e81caa8f.png)
好家伙，好像可以试试这个

**Reply by KaSock** [reply]
> ![image](https://user-images.githubusercontent.com/56071624/123098816-1b9ca180-d464-11eb-850c-0310e81caa8f.png)
> 好家伙，好像可以试试这个

并不行，淦

**Comment by KaSock** [comment]
classes.dex里面就是函数吗...我把里面带ssl删干净了还是叭行...

**Reply by Lost-MSth** [reply]
草，arc是cocos2dx吧，和classes.dex没啥关系（）

**Reply by KaSock** [reply]
> 草，arc是cocos2dx吧，和classes.dex没啥关系（）

啊？杀错了？

**Reply by KaSock** [reply]
> 草，arc是cocos2dx吧，和classes.dex没啥关系（）

教教，如果修改敏感的话，要不咋加个QQ微信啥的也行qwq

**Reply by Lost-MSth** [reply]
嗯，是很敏感，（但更重要的问题是你光是个网络代理转发都搞了好久了，逆向工程要学多久啊（））

**Reply by KaSock** [reply]
> 嗯，是很敏感，（但更重要的问题是你光是个网络代理转发都搞了好久了，逆向工程要学多久啊（））

咳咳，那是意外，这必须交个学费了

**Reply by KaSock** [reply]
开个价吧，一手交q一手教我(

**Reply by Lost-MSth** [reply]
没必要没必要，愿意给就随便给，联系方式发邮箱就行

**Reply by KaSock** [reply]
啊这

**Comment by MBRjun** [comment]
从邮件来的（
新增了多设备自动封号机制 Add multi device auto ban mechanism.
瞧瞧这是人整的吗（
今天上午看了一个登陆4台设备然后封了一周的

**Reply by Lost-MSth** [reply]
最高惊喜封号31天，连arc都不愿意显示了呢~

**Comment by MBRjun** [comment]
/version/database/database_initialize.py:301
这行结尾 '???' 是啥角色（

**Reply by Lost-MSth** [reply]
应该是ns的linka，不过不确定就没写()
